doc/hotspot-style.md

```diff
@@ -572,9 +572,13 @@ There are a few exceptions to this rule.
 
 * `#include <new>` to use placement `new`, `std::nothrow`, and `std::nothrow_t`.
 * `#include <limits>` to use `std::numeric_limits`.
-* `#include <type_traits>`.
+* `#include <type_traits>` with some restrictions, listed below.
 * `#include <cstddef>` to use `std::nullptr_t` and `std::max_align_t`.
 
+Certain restrictions apply to the inclusion of `<type_traits>`.
+
+* The `alignof` operator should always be preferred over `std::alignment_of<>`.
```

Comment:
I would really like to avoid "prefer" here. Even though "always preferred" has
the right meaning, I've always thought that was a clumsy turn of phrase,
unconditionalizing a conditional. Also "prefer over" read oddly to to me. It
seems to be a less commonly used American English usage, with "prefer to"
being more common. (And "prefer to" seems exclusively used in British English,
with "prefer over" not being a thing in that dialect.)

"Use the `alignof` operator rather than `std::alignment_of`."

Reply:
Changes have been made, sorry for the hassle